vcx/dummy-cloud-agent/config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I think this file isn't needed, as it's just a copy of sample-config.json

Reply:
File is removed.